fastlane/lib/fastlane/actions/docs/frame_screenshots.md

```diff
@@ -70,16 +83,69 @@ To download the latest frames
 
 When using _frameit_ without titles on top, the screenshots will have the full resolution, which means they can't be uploaded to the App Store directly. They are supposed to be used for websites, print media and emails. Check out the section below to use the screenshots for the App Store.
 
-# Titles and Background (optional)
+## Advanced Usage (optional)
+
+### Text and Background
 
-With _frameit_ 2.0 you are now able to add a custom background, title and text colors to your screenshots.
+With _frameit_ it's possible to add a custom background and text below or above the framed screenshots in fonts and colors you define.
```

Comment:
I'd use commas to separate what appears to be a list of features: `With _frameit_, it's possible to add a custom background and text, below or above the framed screenshots, in fonts and colors you define.`

Reply:
First one: Nope, as without the text after the comma it would not be a complete sentence any more.
Second and third one: Could do, if "below or above the framed screenshots" should be an addition that could be left off. Here I would be okay with them.

Reply:
For the first one, Grammarly also marked it as an error, but if it makes sense without it too, I have no problem with it.

![screen shot 2018-03-19 at 22 39 44](https://user-images.githubusercontent.com/5748627/37621077-9ed7ee28-2bc6-11e8-99d6-fc813970a11f.png)

Reply:
English is weird.

Reply:
What about:
With _frameit_ it's possible to add a custom background and text below or above the framed screenshots. You can define the text fonts and colors.